### ISSUE TITLE
Error: this.log is undefined

### ISSUE THREAD

**Issue by chasm** [willclarktech/apollo-redux-server#1]
I get this when I try to run the app:

```sh
$ yarn start
yarn start v0.21.3
$ nodemon src/index.js --watch src --exec babel-node
[nodemon] 1.11.0
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: /Users/aries/Workspace/charity/apollo-redux-server/src/**/*
[nodemon] starting `babel-node src/index.js`
/Users/aries/.node/lib/node_modules/babel-register/node_modules/babel-core/lib/transformation/file/options/option-manager.js:266
      this.log.error("Invalid options type for " + alias, TypeError);
              ^

TypeError: Cannot read property 'error' of undefined
    at OptionManager.mergeOptions (/Users/aries/.node/lib/node_modules/babel-register/node_modules/babel-core/lib/transformation/file/options/option-manager.js:266:15)
    at /Users/aries/.node/lib/node_modules/babel-register/node_modules/babel-core/lib/transformation/file/options/option-manager.js:371:14
    at /Users/aries/.node/lib/node_modules/babel-register/node_modules/babel-core/lib/transformation/file/options/option-manager.js:392:24
    at Array.map (native)
    at OptionManager.resolvePresets (/Users/aries/.node/lib/node_modules/babel-register/node_modules/babel-core/lib/transformation/file/options/option-manager.js:387:20)
    at OptionManager.mergePresets (/Users/aries/.node/lib/node_modules/babel-register/node_modules/babel-core/lib/transformation/file/options/option-manager.js:370:10)
    at OptionManager.mergeOptions (/Users/aries/.node/lib/node_modules/babel-register/node_modules/babel-core/lib/transformation/file/options/option-manager.js:330:14)
    at OptionManager.addConfig (/Users/aries/.node/lib/node_modules/babel-register/node_modules/babel-core/lib/transformation/file/options/option-manager.js:232:10)
    at OptionManager.findConfigs (/Users/aries/.node/lib/node_modules/babel-register/node_modules/babel-core/lib/transformation/file/options/option-manager.js:436:16)
    at OptionManager.init (/Users/aries/.node/lib/node_modules/babel-register/node_modules/babel-core/lib/transformation/file/options/option-manager.js:484:12)
[nodemon] app crashed - waiting for file changes before starting...
```

**Comment by willclarktech** [comment]
Hey, thanks a lot for reporting. Which version of `babel-node` are you working with?

**Comment by chasm** [comment]
$ babel-node --version
6.10.1

**Comment by chasm** [comment]
Heh. Realized that I had two versions, one from npm and one from yarn. Deleted the npm one (above) and switched to the global yarn one (6.24.0). Then I had to create a .env file and a logs folder and now it works. Thanks! Looks very cool.

**Comment by willclarktech** [comment]
Glad to hear it! I've added an example `.env` file, and the `postinstall` step now makes a `logs` dir if necessary. Always interested in feedback or ideas, by the way.

**Comment by chasm** [comment]
Nice. Haven't had a chance to play with it yet, but will let you know when I do.

I did the tutorial you based it on way back when it first came out -- one of the best early redux tutorials. Keen to revisit it now that I'm more experienced with redux.

**Comment by willclarktech** [comment]
Reopening to remind me to add `babel-cli` to dependencies.
https://babeljs.io/docs/usage/cli/#installation

**Comment by willclarktech** [comment]
Closed in d0171493bbb15a3a45ea92b34c41c581b9f4645e.
